docs/troubleshooting.md

```diff
@@ -68,6 +68,37 @@ The Kubernetes team hangs out on IRC at [`#google-containers`](https://botbot.me
 
 The Kubernetes mailing list is google-containers@googlegroups.com
 
+# Getting help
+
+If your problem isn't answered by any of guides above, there are variety of ways for you to get help from the Kubernetes team.
```

Comment:
Actually, it looks like the doc got a bit screwed up. Some of the sections are duplicated, like "Getting help" and "Help! My question isn't covered! I need help now!"